01-data-structures/01-introduction-to-data-structures/line/line.rb

```diff
@@ -9,26 +9,36 @@ def initialize
   end
 
   def join(person)
+    self.members.push(person)
   end
 
   def leave(person)
+    self.members.delete(person)
   end
 
   def front
+    self.members[0]
```

Comment:
Ahh, I was referring that _externally_ I wasn't indexing by numbers.

That is, on creation of the object, it would just reference the names, not call by numbers. 

Internally (under the hood), yes, it does.

In this case particularly, it seemed easiest to call the first element like this.